big_tests/tests/push_http_SUITE.erl

```diff
@@ -0,0 +1,279 @@
+-module(push_http_SUITE).
+-compile(export_all).
+
+-include_lib("exml/include/exml.hrl").
+-include_lib("escalus/include/escalus.hrl").
+-include_lib("common_test/include/ct.hrl").
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("escalus/include/escalus_xmlns.hrl").
+
+
+%%--------------------------------------------------------------------
+%% Suite configuration
+%%--------------------------------------------------------------------
+
+all() ->
+    [
+        {group, single},
+        {group, customised},
+        {group, multiple}
+    ].
+
+groups() ->
+    [{single, [sequence], [simple_push]},
+     {customised, [sequence], [custom_push]},
+     {multiple, [sequence], [push_to_many]}
+    ].
+
+suite() ->
```

Comment:
`suite/0` should also `distributed_helper:require_rpc_nodes([mim])`, since the `rpc/3` local helper (line 236) uses `distributed_helper:mim/0`. If this suite is run in isolation, i.e. with no suites already requiring the `mim` node, it will likely fail.